perun-core/src/test/java/cz/metacentrum/perun/core/entry/ConsentsManagerEntryIntegrationTest.java

```diff
@@ -197,6 +197,21 @@ public void getConsentsForUser() throws Exception {
 		assertEquals(0, consentsManagerEntry.getConsentsForUser(sess, user.getId(), ConsentStatus.GRANTED).size());
 	}
 
+	@Test
+	public void getConsentsForConsentHubByResource() throws Exception {
```

Comment:
I would rather have this test create a resource under a facility, assign some members to it, create consents for them and then assert that the method only returns the consents of these assigned members.